cfspecification/cfspecification_test.go

```diff
@@ -10,7 +10,7 @@ var specification Specification
 
 func setup() {
 	var err error
-	specification, err = GetSpecificationFromFile("specification/test_specification.json")
```

Comment:
Actually I liked that - I think it is a good idea to split testing assets from tests.